skyportal/handlers/api/news_feed.py

```diff
@@ -100,6 +100,23 @@ def fetch_newest(model, include_bot_comments=False):
 
                 return newest
 
+            def latest_classification(obj):
```

Comment:
Nice, I would just wrap the whole logic (right after `classification = None`) in a try except with a pass, so it returns no classification instead of an error and the rest of the code can work just fine.